src/backend/utils/adt/jsonpath.c

```diff
@@ -440,13 +440,13 @@ alignStringInfoInt(StringInfo buf)
 	{
 		case 3:
 			appendStringInfoCharMacro(buf, 0);
-			/* FALLTHROUGH */
+			FALL_THROUGH
 		case 2:
 			appendStringInfoCharMacro(buf, 0);
-			/* FALLTHROUGH */
+			FALL_THROUGH
 		case 1:
 			appendStringInfoCharMacro(buf, 0);
-			/* FALLTHROUGH */
+			FALL_THROUGH
```

Comment:
I suggest to remove `default: break;` and don't add `FALL_THROUGH`. The code will be simpler

Reply:
Done